openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -579,25 +580,22 @@ def build_anim(project_name, asset_name):
         project_name, layout_repre, "LinkLayoutLoader"
     )
 
-    # keep layout root collection name before make container publishable.
-    layout_collection_name = next(
-        (
-            root.name
-            for root in layout_container.get_root_datablocks(
-                bpy.types.Collection
-            )
-        ),
-        None,
-    )
     # Make container publishable, expose its content
     bpy.ops.scene.make_container_publishable(
         container_name=layout_container.name
     )
 
-    # Switch hero containers to versioned
+    update_scene_containers()
+
+    # Switch hero containers to versioned and linked setdress to appended
     errors = []
     for container in bpy.context.scene.openpype_containers:
         container_metadata = container["avalon"]
+        family = container_metadata.get("family")
+
+        if family not in ("rig", "model", "setdress"):
```

Comment:
Inclusion testing is faster using a set `{...}`

Reply:
```suggestion
        if family not in {"rig", "model", "setdress"}:
```